core/models/centre.py

```diff
@@ -167,3 +167,15 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class DepartmentalTeam(models.Model):
```

Comment:
There seem to be no tests for the expected behavior of the `DepartmentalTeam` model.

Reply:
This have been added